qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, name, instrument, **kwargs):
         super().__init__(name, names=('',), shapes=((1,),), **kwargs)
         self._instrument = instrument
 
+        print('We rock out at git!')
```

Comment:
@giulioungaretti Boy better know! But actually, Jana promised to practice her git-fu by removing it before anyone saw it...